packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -512,10 +512,19 @@ async function fluid({ file, args = {}, reporter, cache }) {
     size => size < (fixedDimension === `maxWidth` ? width : height)
   )
 
-  // Add the original image to ensure the largest image possible
-  // is available for small images. Also so we can link to
-  // the original image.
-  filteredSizes.push(fixedDimension === `maxWidth` ? width : height)
+  const srcSetAlwaysIncludeOriginal =
+    options.srcSetAlwaysIncludeOriginal === undefined
+      ? true
+      : options.srcSetAlwaysIncludeOriginal
```

Comment:
```suggestion
  const srcSetAlwaysIncludeOriginal = options.srcSetAlwaysIncludeOriginal ?? true
```
Perfect time to use nullish coalescing